### ISSUE TITLE
Questions about TE-DKNet

### ISSUE THREAD

**Issue by chenzean** [huzexi/DKNet#2]
Hello, author. I am interested in TE-DKNet in the paper you wrote, but I can't find the VGG loss function part of TE-DKNet in this repository. I wonder if you can share the VGG loss function section and the full train file? Thank you very much ！！！！
